moralization/spacy_model_manager.py

```diff
@@ -156,19 +156,23 @@ def train(
         data_manager: DataManager,
         use_gpu: int = -1,
         overrides: Optional[Dict] = None,
+        check_data_integrity=True,
     ):
         """Train the model on the data contained in `data_manager`.
 
         Args:
             data_manager (DataManager): the DataManager that contains the training data
             use_gpu (int): The index of the GPU to use (default: -1 which means no GPU)
             overrides (dict): An optional dictionary of parameters to override in the model config
+            check_data_integrity (bool): Wether or not to test the data integrity.
```

Comment:
```suggestion
            check_data_integrity (bool): Whether or not to test the data integrity.
```
![image](https://user-images.githubusercontent.com/12845624/229111898-ece6e340-585d-4375-bb25-f8ad09482dcf.png)